kubernetes_platform/python/kfp/kubernetes/image.py

```diff
@@ -38,11 +38,33 @@ def set_image_pull_secrets(
 
     # Assuming secret_names is a list of strings
     image_pull_secret = [
-        pb.ImagePullSecret(secret_name=secret_name) for secret_name in secret_names
+        pb.ImagePullSecret(secret_name=secret_name)
+        for secret_name in secret_names
     ]
 
     msg.image_pull_secret.extend(image_pull_secret)
 
     task.platform_config['kubernetes'] = json_format.MessageToDict(msg)
 
     return task
+
+
+def set_image_pull_policy(task: PipelineTask, policy: str) -> PipelineTask:
+    """Set image pull policy for the container.
+
+    Args:
+        task: Pipeline task.
+        policy: One of `Always`, `Never`, `IfNotPresent`.
```

Comment:
nit: using enum could be less error-prone, and also works with IDE code auto-complete. No strong preference though.